docs/diagrams/EditTaskCommandSequence.puml

```diff
@@ -0,0 +1,91 @@
+@startuml
```

Comment:
the diagram has too much details i think
could you simplify it? i'm worried it's not very readable when on the DG